app/src/main/java/com/coveros/coverosmobileapp/blogpost/BlogPostsListActivity.java

```diff
@@ -163,16 +154,16 @@ public void onSuccess(List<BlogPost> newPosts) {
     protected void retrieveAuthors(final PostListCallback<String> postListCallback) {
         StringRequest authorsRequest = new StringRequest(Request.Method.GET, AUTHORS_URL, new Response.Listener<String>() {
             @Override
-            public void onResponse(String response) {
-                JsonArray authorsJson = new JsonParser().parse(response).getAsJsonArray();
-                for (JsonElement author : authorsJson) {
-                    JsonObject authorJson = (JsonObject) author;
-                    Integer id = authorJson.get("id").getAsInt();
```

Comment:
Feels like there should maybe be an "Author" object that takes care of all the JSON parsing. similar to "Comment" and BlogPost